src/lib.rs

```diff
@@ -55,7 +55,7 @@
 //!
 //! ## XDG Desktop Portal backend
 //! The XDG Desktop Portal backend is used when the `gtk3` feature is disabled with
-//! [`default-features = false`](https://doc.rust-lang.org/cargo/reference/features.html#dependency-features), and `xdg-portal` is enabled instead. This backend will use either the GTK or KDE file dialog depending on the desktop environment
+//! [`default-features = false`](https://doc.rust-lang.org/cargo/reference/features.html#dependency-features), and `xdg-portal` is enabled instead. Either the `tokio` or `async-std` feature must be enabled. This backend will use either the GTK or KDE file dialog depending on the desktop environment
```

Comment:
Can we enforce this in build.rs?
The same way we enforce selection of gtk3/portal backends.

The CI already failed on confusing error logs from ashpd, so a user friendly error from build.rs would be nice

Reply:
Done.